Modules/Map/QuestieMap.lua

```diff
@@ -359,6 +358,13 @@ function QuestieMap:ShowNPC(npcID, icon, scale, title, body, disableShiftToRemov
     data.IsObjectiveNote = false
     data.ManualTooltipData = {}
     data.ManualTooltipData.Title = title or (npc.name .. " (" .. l10n("NPC") .. ")")
+    local WeaponMasterSkills = require("WeaponMasterSkills")
```

Comment:
Use `QuestieLoader:ImportModule` on top of the file instead of inline `require`.

Reply:
Fixed in: https://github.com/Questie/Questie/pull/5568/commits/9e516ebf1afbac78459a49a1d8fe7b26c0f1dcf1